src/Framework/MockObject/Runtime/ReturnValueGenerator.php

```diff
@@ -57,59 +57,59 @@ public function generate(string $className, string $methodName, string $stubClas
             $types = [$returnType];
         }
 
-        $types = array_map('strtolower', $types);
```

Comment:
strtolower on class-names lead to composer autoloader not beeing able to autoload the classes

Reply:
Cherry-picked into `10.5` and merged to `11.0` and `main` from there.